cli/Valet/Pecl.php

```diff
@@ -56,14 +56,15 @@ class Pecl extends AbstractPecl
             '5.6' => '4.0.11',
             'extension_type' => self::NORMAL_EXTENSION_TYPE
         ],
-        self::GEOIP_EXTENSION => [
-            '7.4' => '1.1.1',
-            '7.3' => '1.1.1',
-            '7.2' => '1.1.1',
-            '7.1' => '1.1.1',
-            '7.0' => '1.1.1',
-            'extension_type' => self::NORMAL_EXTENSION_TYPE
-        ],
+        // self::GEOIP_EXTENSION => [
```

Comment:
Any reason for disabling this?

Reply:
I had alot of issues getting GEOIP to install on m1 mac, and as it wasn't required by valet I commented it out for now.